synapse/rest/client/v2_alpha/account.py

```diff
@@ -542,8 +554,8 @@ class MsisdnThreepidRequestTokenRestServlet(RestServlet):
     def __init__(self, hs):
         self.hs = hs
         super(MsisdnThreepidRequestTokenRestServlet, self).__init__()
-        self.identity_handler = hs.get_handlers().identity_handler
         self.store = self.hs.get_datastore()
+        self.identity_handler = hs.get_handlers().identity_handler
```

Comment:
nit: I'd prefer it if we could avoid no-op changes like this. It makes `git blame` less useful, and increases the likelihood of conflicts.

Reply:
Good point, this is silly.